project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,492 @@
+              - Insulation Ceiling|Uninsulated
+              - Insulation Ceiling|R-7
+              - Insulation Ceiling|R-13
+        costs:
+          - value: 0.0443 # $/(ft^2 * Delta R-value), from original REMDB xlsx for blown cellulose (0.0154+0.019), with 1.289 inflation factor
+            multiplier: Floor Area, Attic * Insulation Increase (ft^2 * Delta R-value)
+          - value: 0.1992 # $/(ft^2), from original REMDB xlsx for blown cellulose (0.1505+0.004), with 1.289 inflation factor
+            multiplier: Floor Area, Attic (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf 
+      - &attic_insulation_IECC_CZ2A-2B-3A-3B-3C
+        option: Insulation Ceiling|R-49
+        apply_logic:
+          - and:
+            - or:
+              - ASHRAE IECC Climate Zone 2004|2A
+              - ASHRAE IECC Climate Zone 2004|2B
+              - ASHRAE IECC Climate Zone 2004|3A
+              - ASHRAE IECC Climate Zone 2004|3B
+              - ASHRAE IECC Climate Zone 2004|3C
+            - Geometry Attic Type|Vented Attic
+            - or:
+              - Insulation Ceiling|Uninsulated
+              - Insulation Ceiling|R-7
+              - Insulation Ceiling|R-13
+              - Insulation Ceiling|R-19
+              - Insulation Ceiling|R-30
+        costs:
+          - value: 0.0443 # $/(ft^2 * Delta R-value), from original REMDB xlsx for blown cellulose (0.0154+0.019), with 1.289 inflation factor
+            multiplier: Floor Area, Attic * Insulation Increase (ft^2 * Delta R-value)
+          - value: 0.1992 # $/(ft^2), from original REMDB xlsx for blown cellulose (0.1505+0.004), with 1.289 inflation factor
+            multiplier: Floor Area, Attic (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      - &attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
+        option: Insulation Ceiling|R-60
+        apply_logic:
+          - and:
+            - or:
+              - ASHRAE IECC Climate Zone 2004|4A
+              - ASHRAE IECC Climate Zone 2004|4B
+              - ASHRAE IECC Climate Zone 2004|4C
+              - ASHRAE IECC Climate Zone 2004|5A
+              - ASHRAE IECC Climate Zone 2004|5B
+              - ASHRAE IECC Climate Zone 2004|6A
+              - ASHRAE IECC Climate Zone 2004|6B
+              - ASHRAE IECC Climate Zone 2004|7A
+              - ASHRAE IECC Climate Zone 2004|7B
+            - Geometry Attic Type|Vented Attic
+            - or:
+              - Insulation Ceiling|Uninsulated
+              - Insulation Ceiling|R-7
+              - Insulation Ceiling|R-13
+              - Insulation Ceiling|R-19
+              - Insulation Ceiling|R-30
+              - Insulation Ceiling|R-38
```

Comment:
missing `- Insulation Ceiling|R-49`, is this intentional?

Reply:
Yes, it is intentional.
The intent document is to apply to dwelling units with R-38 or less insulation. I refer to the previous EUSS run. I'm not sure whether it is correct.